src/lib/messenger.js

```diff
@@ -34,7 +47,7 @@ export default function messenger(bot) {
       } else if (inputParser.isAskingForBoegivar(text)) {
         handleBoegivar(bot, message.chat, user.firstName);
       } else if (inputParser.isAskingForTourettes(text)) {
-        handleTourettes(bot, message.chat, text);
```

Comment:
=) you read my mind! That what I was going for at first when the original striptext returned command, text, and a boolean to check if the text had a command to begin with. How about you make an issue and we can fiddle with it after this version goes live and we all fork from there?